app/views/dispatch/_messages.html.haml

```diff
@@ -1,4 +1,6 @@
-- if (@conversation.user.phone_number_normalized.present? && !@conversation.messages.empty?)
+-# Mashing haml around with js is not the way to do this.
+
+- if (@conversation&.user&.phone_number_normalized.present? && !@conversation.messages.empty?)
```

Comment:
`&& !@conversation.messages.empty?` might be more readable as `&& @conversation.messages.any?`

Reply:
👍